mlflow/pytorch/torchserve/deploy/serve.py

```diff
@@ -0,0 +1,388 @@
+                                if PurePath(sub_files).name == requirements_file:
+                                    req_file_path = sub_files
+                        else:
+                            if file.suffix in valid_file_suffixes:
+                                model_path = file
+                            if PurePath(file).name == requirements_file:
+                                req_file_path = file
+                    if model_path is None:
+                        raise RuntimeError(
+                            "Model file does not have a valid suffix. Expected to be one of "
+                            + ", ".join(valid_file_suffixes)
+                        )
+                    model_uri = model_path
+
+        export_path = self.server_config["export_path"]
+        if export_path:
+            model_store = export_path
+        else:
+            model_store = "model_store"
+            if not os.path.isdir(model_store):
+                os.makedirs(model_store)
+
+        cmd = (
+            "torch-model-archiver --force --model-name {} "
+            "--version {} --model-file {} --serialized-file {} "
+            "--handler {} --export-path {}".format(
+                model_name, version, model_file, model_uri, handler_file, model_store
+            )
+        )
+        if extra_files:
+            cmd = "{} --extra-files {}".format(cmd, extra_files)
+        if req_file_path:
+            cmd = "{} -r {}".format(cmd, req_file_path)
+
+        return_code = os.system(cmd)
+        if return_code != 0:
+            _logger.error(
+                "Error when attempting to load and parse JSON cluster spec from file %s",
+                cmd,
+            )
+            raise Exception("Unable to create mar file")
+
+        if export_path:
+            mar_file = "{}/{}.mar".format(export_path, model_name)
+        else:
+            mar_file = "{}/{}/{}.mar".format(os.getcwd(), model_store, model_name)
+            if os.path.isfile(mar_file):
+                print("{} file generated successfully".format(mar_file))
+        return mar_file
+
+    def __register_model(
+        self, mar_file_path, name, model_uri=None, flavor=None, config=None
+    ):
+        """
+        Register the model using the mar file that has been generated by the archiver
+        """
+        query_path = mar_file_path
+        if config:
+            for key in config:
+                query_path += "&" + key + "=" + str(config[key])
+        else:
+            query_path += "&initial_workers=" + str(1)
+
+        url = "{}/{}?url={}".format(self.management_api, "models", query_path)
+        resp = requests.post(url=url)
+
+        if resp.status_code != 200:
+            raise Exception("Unable to register the model")
+        return True
+
+
+def run_local(name, model_uri, flavor=None, config=None):
+    device = config.get("device", "cpu")
+    if "gpu" in device.lower():
```

Comment:
```suggestion
    if device.lower() == "gpu":
```

Is there any reason not to check exact equality here?